.github/workflows/publish-maven-snapshots.yml

```diff
@@ -5,8 +5,9 @@ on:
   push:
       branches:
         - main
-        - '1.3'
+        - 1.3
         - 2.x
+        - 2.8
```

Comment:
@gaiksaya 1.3 is fine, but for every new 2.x release would need make a changes once again (2.9, ...), I bilieve @cwperks origin approach was the right one.

Reply:
I also prefer the original approach - what is the risk if a snapshot artifact in an old published branch is out-of-sync with the published artifacts? Its pretty rare that changes are merged into branches that have been published except for the most recent published version.

Reply:
I agree, yet another benefit

Reply:
Works for me. I received some push back in my previous PR. If no concerns by the maintainers we can move forward as is.